test/model/test_datatypes.py

```diff
@@ -7,6 +7,7 @@
 import datetime
 import math
 import unittest
+from builtins import ValueError
```

Comment:
You don't have to import `ValueError` here as it's a builtin and available by default.

Reply:
I removed the ValueError import. All tests ran fine.